app/experimenter/nimbus-ui/src/components/Summary/TableSummary/index.test.tsx

```diff
@@ -126,7 +126,7 @@ describe("TableSummary", () => {
     });
   });
 
-  describe("renders 'Risk mitigation checklist' row as expected", () => {
+  describe("renders 'Risk mitigation checklist (old)' row as expected", () => {
```

Comment:
Hmm, thought maybe we could just remove this test altogether. But I see why it's here. Might make sense to rephrase this as "obsolete implementation" rather than "old" if only because I scratched my head for a moment

Reply:
Oh oops, I forgot to remove that, I originally changed the row in the Summary component to also say (old) and then chatted with Ana this morning and we decided it's not needed. I think "obsolete implementation" is better for a test descriptor anyways, I'll change that when I fix anything else. We can remove this test when #5172 is done. :)